Update Collections governance (#112)
OpenTermsArchive/docs: content/collections/create.en.md

# Creating a collection

You are considering creating a new collection to track terms with Open Terms Archive? Amazing!

## Define metadata

First of all, define the [metadata]({{< relref "collections/metadata" >}}) of the collection you would like to create.

## Check existing collections

Now that you have a clear idea what you would like to track, double-check that there are no [existing federated collections](https://opentermsarchive.org/#collections) that you could contribute to. If you have a doubt about whether some terms you want to track would fit a collection, reach out to the collection maintainers.

> If no existing collection could be a good host for the terms you would like to track, then it is relevant to create your own.

Reviewer: This explanation about checking existing collections doesn't suggest that creating similar collections is a good thing, although I think it's a good idea to encourage it.

Reviewer 2: As maintenance represents a large part of the work and not just the creation of the collection, I think we should encourage firstly contributing to an existing collection to increase the chances of having quality datasets and only secondly, we should encourage creating a collection with similar services, which increase overall resilience.

## Inform the community

Starting a new collection is an exciting endeavour, and would strongly benefit from the support of the community who already maintains existing collections. It is strongly recommended to share your intention to create a new collection as early as possible in the process, to get support and identify potential partners.

> You can inform the community by posting on the instant messaging system, or [sending an email](mailto:contact@opentermsarchive.org) to the core team.

Reviewer: Don't we offer a direct link to Mattermost?

Reviewer 2: We could explain that if she/he wants to join the instant messaging system, she/he should make a request to us.

> You can inform the community by posting on the instant messaging system, or [sending an email](mailto:contact@opentermsarchive.org) to the core team.

Reviewer: ```suggestion
[Join the community]({{< relref "collections/federation#how-to-join" >}}) to inform the community by posting on the instant messaging system, or [sending an email](mailto:contact@opentermsarchive.org) to the core team.
```
